templates/github/.github/workflows/ci.yml.j2

```diff
@@ -136,7 +136,7 @@ jobs:
         shell: bash
 
       - name: After failure
-        if: {{ "${{ failure() }}" }} 
+        if: failure()
```

Comment:
This seems right to me but it also seems that this was recently changed?

https://github.com/pulp/plugin_template/pull/435

I guess ttereshc mistakenly thought the after failure step wasn't being run for nightly because of the formatting?

Reply:
That was my guess also